### ISSUE TITLE
Dev utility: Add or update new package

### ISSUE THREAD

**Issue by pombredanne** [nexB/scancode-toolkit#2088]
As a developer of ScanCode, I want to update some of the dependent packages or add a new package.

For that I somehow fetch some wheels and/or tarballs, craft ABOUT files, collect licenses and store these in a local directory (for instance thirdparty/) which is not meant to be committed.
I may also update my setup.py with new version ranges or constraints or I may add a new package.

Once this is done, I want to call a command line tool, such that an existing requirements file will be updated with these new and updated packages.

Next these may need to be uploaded to our PyPI... see Upload to GitHub release #2136 for that

**Comment by pombredanne** [comment]
This is a duplicate of two tickets:
1. Dev utility: build and download the sdist and wheels of a single package and all its dependencies nexB/scancode-toolkit#2087
2. Dev utility: Create script to generate requirements with hashes nexB/skeleton#50 
Therefore it is being closed in favor these other tickets.

**Comment by pombredanne** [comment]
Eventually we built a tool that handles creation, documentation and publishing of wheels and source https://github.com/nexB/scancode-toolkit/tree/1bddb92c70abcf26cf9435061e429bae62cbe040/etc/release
And this is also part of the https://github.com/nexB/skeleton

Using romp it builds wheel and tests them on many OS/Python combos

In the end they end up pushed to https://thirdparty.aboutcode.org/pypi/
